plugins/net_plugin/net_plugin.cpp

```diff
@@ -1197,7 +1228,7 @@ namespace eosio {
          auto ds = pending_message_buffer.create_datastream();
          net_message msg;
          fc::raw::unpack(ds, msg);
-         msgHandler m(impl, shared_from_this() );
+         msg_handler m(impl, shared_from_this() );
          msg.visit(m);
```

Comment:
If `msg` contains `signed_block`, this will require special case logic to move it and pass that rvalue reference to the function call operator of the `m` visitor. Otherwise, it could use the regular `visit` method of `msg`.